docusaurus.config.ts

```diff
@@ -97,6 +97,12 @@ const config: Config = {
               label: "Tutorial",
               to: "/docs/tutorial/getting-started",
             },
+            {
+              html: `<a href="/pydocs/index.html" target="_blank" rel="noopener noreferrer" style="text-decoration: none;">Python API References</a>`,
+            },
+            {
+              html: `<a href="/tsdocs/index.html" target="_blank" rel="noopener noreferrer" style="text-decoration: none;">Javascript(Node) API References</a>`,
+            },
```

Comment:
<img width="301" alt="image" src="https://github.com/user-attachments/assets/169c32e0-dd47-4992-8d81-8c50d50a0800" />

지금은 localhost라 적용이 힘들겠지만, deploy하고나면 `https://` 부터해서 full url을 넣어서 할 수 있을 것 같습니다. 일단 이대로 두시죠.

```suggestion
            {
              label: "Python API References",
              href: "http://localhost:3000/pydocs/index.html",
            },
            {
              label: "Javascript(Node) API References",
              href: "http://localhost:3000/tsdocs/index.html",
            },
```